### ISSUE TITLE
Installation failed

### ISSUE THREAD

**Issue by OneSheep99** [pail23/stiebel_eltron_isg_component#9]
Dear Paul, 
I have followed your instructions using the latest version from git, but I have run into some issues when adding a device. I get an error that the config flow is broken. In the log file I get:

`2023-01-05 20:23:15.770 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration stiebel_eltron_isg which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant`

`2023-01-05 20:27:02.211 ERROR (MainThread) [homeassistant.config_entries] Error occurred loading configuration flow for integration stiebel_eltron_isg: No module named 'pymodbus.client3`

## Describe the bug
Whenever I try to add an device I get an error, that the config flow is interrupted. 
I'm using 
```
Home Assistant 2023.1.0
Supervisor 2022.12.1
Operating System 9.4
Frontend 20230104.0 - latest
````

thanks a lot,
Kai

**Comment by pail23** [comment]
This should be fixed with the new release 0.1.0. Could you please check if it is working now?

**Comment by OneSheep99** [comment]
The fix did it! Thanks a lot for the quick response!
